internal/controllers/nodequotaconfig_controller.go

```diff
@@ -98,6 +98,8 @@ func (r *NodeQuotaConfigReconciler) SetupWithManager(mgr ctrl.Manager) error {
 // CalculateRootSubnamespaces calculates the resource allocation for the root subnamespaces based on the provided NodeQuotaConfig.
 // It takes a context, the NodeQuotaConfig to reconcile, and a logger for logging informational messages.
 // It returns an error (if any occurred) during the calculation.
+// If an error occurs during the updating of the root subnamespaces, it returns false but continues.
```

Comment:
// If an error occurs during the updating of the root subnamespaces, it logs the error but continues;
// This is due to the fact that many times, errors occur because nodes have been added or removed from the cluster since the last calculation.